### PR TITLE
German translation fix

### PR DESCRIPTION
**Pull request by mairdl**
The German translation of JabRef has some issues, have a look at the following screenshot:

file-open

The function to open a database is called `Datei öffnen` the displayed hotkey is `F4`. In the English version of JabRef the file menu looks like this:

openfilemenu

As you can see the right hotkey for this function should be` CTRL + O`. Also, the word database gets translated as `Datei` instead of `Datenbank`.

This PR aims to fix this issue by rewording the translation. Every English occurrence of `database `is now translated as `Datenbank`. This is what it looks like now:

openfilemenu

- [x] Change in CHANGELOG.md described
- [x] Manually tested changed features in running JabRef
- [x] internal qs

### CODE REVIEW

**Comment by Siedlerchr** [comment]
> This PR aims to fix this issue by rewording the translation. Every English occurrence of databaseis now translated as Datenbank

I doubt that this is a good idea! 
We differ between a Bibtex-Database (Datei (because it is not a real Datenbank) and a real SQL-Database (Datenbank). The question is more likely: Why is ist called "database" in english? 
And the answer is possibly Historic reason...
@koppor

**Comment by koppor** [comment]
I discussed with @mairdl and others here. The English menu strings are the leading strings. And we currently have "Database" there.

I know that this is confusing. This also confused me multiple times.

What @mairdl didn't show that we also have a Tools menu with "Open file". There, the attached (!) file is shown. We thought that is is a good idea to distinguish between (linked/attached) files (== files) and bib files (== database).

For now, I would merge this in. Otherwise we have to change **all** translations.

**Comment by mairdl** [comment]
A screenshot of the `Tools/Extra menu`:

toolmenu

To open an attached file you select `Datei öffnen`  same as to open a new database.

File: src/main/resources/l10n/JabRef_de.properties [hunk 1]
```diff
@@ -1611,7 +1611,7 @@ The_paragraph_format_is_controlled_by_the_property_'ReferenceParagraphFormat'_or
 The_character_format_is_controlled_by_the_citation_property_'CitationCharacterFormat'_in_the_style_file.=Das_Zeichenformat_wird_von_der_Eigenschaft_'CitationCharacterFormat'_in_der_Stildatei_bestimmt.
 Automatically_sync_bibliography_when_inserting_citations=Bibliographie_beim_Einfügen_von_Literaturverweisen_automatisch_synchronisieren
 Look_up_BibTeX_entries_in_the_active_tab_only=BibTeX-Einträge_nur_im_aktiven_Tab_suchen
-Look_up_BibTeX_entries_in_all_open_databases=BibTeX-Einträge_in_allen_geöffneten_Dateien_suchen
+Look_up_BibTeX_entries_in_all_open_databases=BibTeX-Einträge_in_allen_geöffneten_Datenbank_suchen
```
Comment (tschechlovdev) [inline src/main/resources/l10n/JabRef_de.properties]: Datenbanken

File: src/main/resources/l10n/JabRef_de.properties [hunk 2]
```diff
@@ -1639,8 +1639,8 @@ JabRef_includes_a_built-in_list_of_journal_abbreviations.=JabRef_verfügt_über_
...
-This_feature_generates_a_new_database_based_on_which_entries_are_needed_in_an_existing_LaTeX_document.=Diese_Funktion_erstellt_eine_neue_Datei_basierend_auf_den_Einträgen,_die_von_einem_bestehenden_LaTeX-Dokument_benötigt_werden.
-You_need_to_select_one_of_your_open_databases_from_which_to_choose_entries,_as_well_as_the_AUX_file_produced_by_LaTeX_when_compiling_your_document.=Sie_müssen_eine_Ihrer_geöffneten_Dateien,_von_denen_Einträge_genommen_werden_sollen,_sowie_die_AUX-Datei,_die_von_LaTeX_beim_Kompilieren_Ihres_Dokuments_erstellt_wird,_auswählen.
+This_feature_generates_a_new_database_based_on_which_entries_are_needed_in_an_existing_LaTeX_document.=Diese_Funktion_erstellt_eine_neue_Datenbank_basierend_auf_den_Einträgen,_die_von_einem_bestehenden_LaTeX-Dokument_benötigt_werden.
+You_need_to_select_one_of_your_open_databases_from_which_to_choose_entries,_as_well_as_the_AUX_file_produced_by_LaTeX_when_compiling_your_document.=Sie_müssen_eine_Ihrer_geöffneten_Datenbank,_von_denen_Einträge_genommen_werden_sollen,_sowie_die_AUX-Datei,_die_von_LaTeX_beim_Kompilieren_Ihres_Dokuments_erstellt_wird,_auswählen.
```
Comment (tschechlovdev) [inline src/main/resources/l10n/JabRef_de.properties]: geöffneten Datenbanken

File: src/main/resources/l10n/Menu_de.properties [hunk 3]
```diff
@@ -44,12 +44,12 @@ New_%0_database=Neue_%0_Datenbank
...
 New_entry_from_plain_text=Neuer_Eintrag_aus_&Klartext
-New_subdatabase_based_on_AUX_file=&Neue_Teildatei_aus_AUX-Datei
+New_subdatabase_based_on_AUX_file=&Neue_Teildatei_aus_AUX-Datenbank
 # View
 Next_tab=&N\u00e4chster_Tab
-Open_database=Datei_\u00f6&ffnen
+Open_database=Datenbank_\u00f6&ffnen
 Open_URL_or_DOI=&URL_oder_DOI_\u00f6ffnen
-Open_shared_database=Entfernte_Datenbak_öffnen
+Open_shared_database=Entfernte_Datenbank_öffnen
```
Comment (tschechlovdev) [inline src/main/resources/l10n/Menu_de.properties]: Why is shared translated with "entfernte" ?

File: src/main/resources/l10n/Menu_de.properties [hunk 4]
```diff
@@ -82,8 +82,8 @@ Mark_specific_color=Mit_bestimmter_&Farbe_markieren
 Unmark_all=S\u00e4m&tliche_Markierungen_aufheben
 Unmark_entries=Markierung_a&ufheben
 View=&Ansicht
-Import_into_new_database=&Importieren_in_neue_Datenbank
-Import_into_current_database=Importieren_in_aktuelle_&Datenbank
+Import_into_new_database=&Importiere_in_neue_Datenbank
+Import_into_current_database=Importiere_in_aktuelle_&Datenbank
```
Comment (tschechlovdev) [inline src/main/resources/l10n/Menu_de.properties]: In my opinion "Importieren" is better than "importiere".

File: src/main/resources/l10n/JabRef_de.properties [hunk 5]
```diff
@@ -1696,7 +1696,7 @@ Select_file_type\:=Dateityp_wählen\:
 These_files_are_not_linked_in_the_active_database.=Diese_Dateien_sind_in_der_aktiven_Datenbank_nicht_verlinkt.
 Entry_type_to_be_created\:=Eintragstyp,_der_erstellt_werden_soll\:
 Searching_file_system...=Dateisystem_wird_durchsucht...
-Importing_into_Database...=Importieren_in_die_Datei...
+Importing_into_Database...=Importieren_in_die_Datenbank...
```
Comment (tschechlovdev) [inline src/main/resources/l10n/JabRef_de.properties]: You should stay consistent here. If you decide for "Importiere" below, then you should also use here "Importiere".

File: src/main/resources/l10n/Menu_de.properties (hunk 4 above)
Comment (mairdl) [inline src/main/resources/l10n/Menu_de.properties]: Ok I will invert it then

File: src/main/resources/l10n/Menu_de.properties (hunk 3 above)
Comment (mairdl) [inline src/main/resources/l10n/Menu_de.properties]: This is a translation of @obraliar as far as I know

Comment (Siedlerchr) [inline src/main/resources/l10n/Menu_de.properties]: I would go for "Geteilte_Datenbank_öffnen"

Comment (tschechlovdev) [inline src/main/resources/l10n/Menu_de.properties]: Yeah, but "Entfernte" is really confusing I think. "Geteilte" sounds good.

File: CHANGELOG.md [hunk 6]
```diff
@@ -35,6 +35,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Implemented integrity check for `note` and `howpublished` field: Should have the first letter capitalized (BibTeX)
 - <kbd>Pos1</kbd> / <kbd>HOME</kbd> now select the first/last entry in the main table and the search result frame.
 - <kbd>UP</kbd> / <kbd>Down</kbd> / <kbd>Tab</kbd> / <kbd>shift+Tab</kbd> in the search result frame have now the same functionality as in the main  table.
+- Generalize german translation of database into "Datenbank"
```
Comment (boceckts) [inline CHANGELOG.md]: Please aslo add a comment in the Fixed section that you fixed the keybinding bug when using the german language.

File: src/main/resources/l10n/Menu_de.properties (hunk 4 above)
Comment (boceckts) [inline src/main/resources/l10n/Menu_de.properties]: Please change it to "Importieren".

File: src/main/resources/l10n/Menu_de.properties (hunk 3 above)
Comment (boceckts) [inline src/main/resources/l10n/Menu_de.properties]: You changed the other strings to "Geteilte" so this needs to be changed, too.

**Comment by tobiasdiez** [comment]
I didn't inspect the changes closely, but how did you fixed the wrong keyboard shortcut? I just saw a few changes in the properties files but nothing related to keyboard shortcuts.

Btw: Zotero, Mendeley and Papers all call it "Library".

**Comment by koppor** [comment]
It "magically" solved the shortcut issue. Maybe the shortcuts are stored using the translate menu items? @Braunch @mairdl Any comments?

If we change from "database" to "library", we have to change all translations everywhere and also the text at help.jabref.org.  Let's discuss this at the dev call.

**Comment by mairdl** [comment]
I think JabRef would assign the wrong keybinding, because of the same german translation for 2 diffrent actions.

File: CHANGELOG.md [hunk 7]
```diff
@@ -38,6 +38,8 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - <kbd>Pos1</kbd> / <kbd>HOME</kbd> now select the first/last entry in the main table and the search result frame.
 - <kbd>UP</kbd> / <kbd>Down</kbd> / <kbd>Tab</kbd> / <kbd>shift+Tab</kbd> in the search result frame have now the same functionality as in the main  table.
 - Importer for MODS format added
+- Generalize german translation of database into "Datenbank"
```
Comment (koppor) [inline CHANGELOG.md]: Typo. `German`

File: CHANGELOG.md (hunk 6 above)
Comment (koppor) [inline CHANGELOG.md]: Please write "German" in with capital `G`

File: CHANGELOG.md [hunk 8]
```diff
@@ -66,6 +68,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed [#1958](https://github.com/JabRef/jabref/issues/1958): Verbatim fields are no longer checked for HTML encoded characters by integrity checks
 - Fixed [#1937](https://github.com/JabRef/jabref/issues/1937): If no help page for the current chosen language exists, the english help page will be shown
 - Fixed [#1996](https://github.com/JabRef/jabref/issues/1996): Unnecessary other fields tab in entry editor removed (BibTeX mode)
+- Fixed file menu displays wrong hotkey in the german translation
```
Comment (koppor) [inline CHANGELOG.md]: `G`erman

File: src/main/resources/l10n/Menu_de.properties [hunk 9]
```diff
@@ -87,7 +87,7 @@ Import_into_current_database=Importieren_in_aktuelle_&Datenbank
 
 Switch_to_%0_mode=In_den_%0-Modus_wechseln
 Push_entries_to_external_application_(%0)=&Eintr\u00e4ge_in_externe_Anwendung_einf\u00fcgen_(%0)
-Pull_changes_from_shared_database=Entfernte_Änderungen_beziehen
+Pull_changes_from_shared_database=\u00c4nderungen_der_geteilten_Datenbank_beziehen
```
Comment (koppor) [inline src/main/resources/l10n/Menu_de.properties]: Do not use `\u00c4`, use the Umlaut directly. We have UTF-8 and this works perfectly.

**Comment by koppor** [comment]
@mairdl Please resolve conflicts, address the minor comments. Then it should good to go

@JabRef/stupro-qa  Could you please double-check the final changes by @mairdl? It seems, the Importiere/Importieren thing could not have been solved consistently.

**Comment by boceckts** [comment]
@mairdl "Datenbank" is still not consistently translated in all strings, e.g. 
Append_database=Datei_anhängen
Automatically_create_groups_for_database.=Automatisch_Gruppen_für_die_Datei_anlegen.
Close_the_current_database=Aktuelle_Datei_schließen
Closed_database=Datei_geschlossen

same goes for "importieren", e.g.
Importing=Importiere
Importing_in_unknown_format=Importiere_ein_unbekanntes_Format
Importing_in_%0_format=Importiere_im_Format_%0

Please use `strg F` in the JabRef_de.properties file to find all occurences

File: src/main/resources/l10n/Menu_de.properties (hunk 9 above)
Comment (mairdl) [inline src/main/resources/l10n/Menu_de.properties]: Sometimes the Umlaut isn't used, i guess i can change all of them

File: src/main/resources/l10n/JabRef_de.properties (hunk 5 above)
Comment (koppor) [inline src/main/resources/l10n/JabRef_de.properties]: Still not fixed.

File: src/main/resources/l10n/JabRef_de.properties [hunk 10]
```diff
@@ -1223,9 +1223,9 @@ Stored_entry=Eintrag_gespeichert
 
 Strings=Ersetzen
 
-Strings_for_database=Strings_für_die_Datei
+Strings_for_database=Strings_für_die_Datenbank
 
-Subdatabase_from_AUX=Teildatenbank_aus_AUX-Datei
+Subdatabase_from_AUX=Teildatenbank_aus_AUX-Datenbank
```
Comment (koppor) [inline src/main/resources/l10n/JabRef_de.properties]: AUX is really a file. Please revert this change.

See https://de.wikibooks.org/wiki/LaTeX/_Grundlagen for further information on AUX files.

**Comment by koppor** [comment]
@mairdl Please fix my comment and `Importing_into_Database...`

File: src/main/resources/l10n/JabRef_de.properties (hunk 10 above)
Comment (mairdl) [inline src/main/resources/l10n/JabRef_de.properties]: I thought because a subdatabase was created from AUX, AUX had to be a database. I will revert it.

**Comment by boceckts** [comment]
Can you please update the screenshots in your description?

File: src/main/resources/l10n/JabRef_de.properties [hunk 11]
```diff
@@ -1693,7 +1693,7 @@ Select_file_type\:=Dateityp_wählen\:
 These_files_are_not_linked_in_the_active_database.=Diese_Dateien_sind_in_der_aktiven_Datenbank_nicht_verlinkt.
 Entry_type_to_be_created\:=Eintragstyp,_der_erstellt_werden_soll\:
 Searching_file_system...=Dateisystem_wird_durchsucht...
-Importing_into_Database...=Importieren_in_die_Datei...
+Importing_into_Database...=In die Datenbank importieren...
```
Comment (boceckts) [inline src/main/resources/l10n/JabRef_de.properties]: Underscores are missing here

File: src/main/resources/l10n/JabRef_de.properties [hunk 12]
```diff
@@ -2283,8 +2283,8 @@ Look_up_full_text_documents=Volltextdokumente suchen
 You_are_about_to_look_up_full_text_documents_for_%0_entries.=Sie werden gleich Volltextdokumente für %0 Einträge suchen
 last_four_nonpunctuation_characters_should_be_numerals=Die letzten vier Nichtinterpunktionszeichen sollten Numerale sein
 shared=geteilt
-should_contain_an_integer_or_a_literal=
-should_have_the_first_letter_capitalized=
+should_contain_an_integer_or_a_literal=Sollte einen Integer oder einen Literal enthalten
+should_have_the_first_letter_capitalized=Sollte den ersten Buchstaben großgeschrieben haben
```
Comment (boceckts) [inline src/main/resources/l10n/JabRef_de.properties]: Not really part of this pr but if you want to translate it please replace spaces by underscores.

File: src/main/resources/l10n/JabRef_de.properties [hunk 13]
```diff
@@ -2294,8 +2294,8 @@ Fetcher_'%0'_did_not_find_an_entry_for_id_'%1'.=Der_Fetcher_%0_hat_keinen_Eintra
...
-Invalid_ISBN\:_'%0'.=
-should_be_an_integer_or_normalized=
-should_be_normalized=
-empty_BibTeX_key=
-BibLaTeX_field_only=
+Invalid_ISBN\:_'%0'.=Ungültige ISBN\:_'%0'.
+should_be_an_integer_or_normalized=Sollte ein Integer oder normalisiert sein
+should_be_normalized=Sollte normalisiert sein
+empty_BibTeX_key=Leerer BibTeX-Key
+BibLaTeX_field_only=Nur ein BibLaTeX-Feld
```
Comment (boceckts) [inline src/main/resources/l10n/JabRef_de.properties]: Same aslast comment.

File: CHANGELOG.md [hunk 14]
```diff
@@ -78,7 +80,9 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed loop when pulling changes (shared database) when current selected field has changed
 - Fixed [#1958](https://github.com/JabRef/jabref/issues/1958): Verbatim fields are no longer checked for HTML encoded characters by integrity checks
 - Fixed [#1937](https://github.com/JabRef/jabref/issues/1937): If no help page for the current chosen language exists, the english help page will be shown
+- Fixed [#1996](https://github.com/JabRef/jabref/issues/1996): Unnecessary other fields tab in entry editor removed (BibTeX mode)
```
Comment (koppor) [inline CHANGELOG.md]: This is not part of this PR, is it?

File: src/main/resources/l10n/Menu_de.properties [hunk 15]
```diff
@@ -2,128 +2,128 @@
...
 Global_Search=Globale_Suche
-Select_all=&Alle_ausw\u00e4hlen
+Select_all=&Alle_auswählen
 Set_up_general_fields=Allgemeine_&Felder_festlegen
 Show_error_console=Fehlerkonsole_anzeigen
 Sort_tabs=Tabs_&sortieren
-Next_preview_layout=
-Previous_preview_layout=
+Next_preview_layout=Nächstes Vorschaulayout
+Previous_preview_layout=Vorheriges Vorschaulayout
```
Comment (boceckts) [inline src/main/resources/l10n/Menu_de.properties]: Same problem with underscores.

File: CHANGELOG.md (hunk 14 above)
Comment (mairdl) [inline CHANGELOG.md]: must have slipped in by mistake

File: src/main/resources/l10n/Menu_de.properties (hunk 9 above)
Comment (Siedlerchr) [inline src/main/resources/l10n/Menu_de.properties]: Yes,! The tests will fail if you use the encoded \uXXX things.  Make sure to replac them all (I had to change it in the Italian lang files)
e

File: CHANGELOG.md (hunk 14 above)
Comment (Siedlerchr) [inline CHANGELOG.md]: Probably because the original was unoded a few days ago,

**Comment by mairdl** [comment]
Travis seems to be unrelated

**Comment by koppor** [comment]
"Zuletzt geöffenete Dateien" still shows "Datei" and not "Database" :suspect:

**Comment by mairdl** [comment]
Well in English it says `recent files` soi I kept it that way. If the key changes i can change it 🎅

**Comment by koppor** [comment]
Why not changing the key (with all consequences), too?

**Comment by mairdl** [comment]
because that is not related to this PR, this would have to be a seperate one (This PR focuses on the German translation, your suggested change would include all language files) I will create a PR for your suggestion. (link will follow soon)
